.gitignore

```diff
@@ -28,16 +28,16 @@ data-plane/**/build/
 **/.vscode/
 
 ## Ignore released file
-eventing-kafka-controller.yaml
-eventing-kafka-source.yaml
-eventing-kafka-broker.yaml
-eventing-kafka-channel.yaml
-eventing-kafka-sink.yaml
-
-eventing-kafka-post-install.yaml
-
-eventing-kafka-controller-prometheus-operator.yaml
-eventing-kafka-source-prometheus-operator.yaml
-eventing-kafka-channel-prometheus-operator.yaml
-eventing-kafka-broker-prometheus-operator.yaml
-eventing-kafka-sink-prometheus-operator.yaml
+/eventing-kafka-controller.yaml
+/eventing-kafka-source.yaml
+/eventing-kafka-broker.yaml
+/eventing-kafka-channel.yaml
+/eventing-kafka-sink.yaml
+
+/eventing-kafka-post-install.yaml
+
+/eventing-kafka-controller-prometheus-operator.yaml
+/eventing-kafka-source-prometheus-operator.yaml
+/eventing-kafka-channel-prometheus-operator.yaml
+/eventing-kafka-broker-prometheus-operator.yaml
+/eventing-kafka-sink-prometheus-operator.yaml
```

Comment:
without `/` it ignores files in `openshift/release/artifacts` with the same name.